Automattic/wp-calypso
# Reader: add following intro component

Now that Cold Start has been turned off, we have no way of introducing new users to the Reader.

This PR adds a dismissible box shown only to new users of the Reader, based on this design:

<img width="789" alt="screen shot 2017-02-08 at 15 24 53" src="https://cloud.githubusercontent.com/assets/17325/22765134/c5abf3e8-ee12-11e6-8d86-f0df36a13914.png">

### To test

Create a new user and verify that you are shown the banner at the top of the Following stream.

Click the close button to dismiss it. Ensure that you are not shown the banner again, either in the same session or after a hard refresh.
